docs/source/app_developers_guide/example_transaction_processors.rst

```diff
@@ -0,0 +1,40 @@
+******************************
+Example Transaction Processors
+******************************
+
+Sawtooth includes several transaction families as examples for developing a
+transaction processor. The following executables are available:
+
+* ``block-info-tp`` - BlockInfo transaction processor (written in Python)
+
+* ``identity-tp`` - Identity transaction processor (written in Python)
+
+* ``intkey-tp-go`` - IntegerKey transaction processor written in Go
+
+* ``intkey-tp-java`` - IntegerKey transaction processor written in Java
+
+* ``intkey-tp-javascript`` - IntegerKey transaction processor written in
+  JavaScript (requires node.js)
```

Comment:
Node.js is typically capitalized.

The "required" wording also feels a little odd to me, since Node is a runtime environment (as opposed to a browser), rather than real a dependency. I think just `(Node.js)`, would be preferable, and is more typical of what you'd see when people are trying to emphasize JS is not meant for the browser.

Reply:
Fixed.